src/compliance-rules.js

```diff
@@ -218,6 +223,37 @@ function addPopover(activityId, bpmnVisualization) {
     trigger: "click",
   });
 
+  // Add mouseover and mouseout event listeners to the table rows
+  tippyInstance.popper.addEventListener("mouseover", (event) => {
+    if (event.target.nodeName === "TD") {
+      const selectedRow = event.target.parentElement;
+      selectedRow.style.backgroundColor = "lightgray"; 
+      const activityName = selectedRow.firstElementChild.textContent;
+      const activityId = getElementIdByName(activityName)
+      //highlight activity
+      bpmnVisualization.bpmnElementsRegistry.addCssClasses(activityId, "cause-violation")
+      // workaround for https://github.com/process-analytics/icpm-demo-2022/issues/87
+      if(complianceData.has(activityName)){
+        addRippleCircles(activityId, bpmnVisualization)
+      }
+    }
+  });
+
+  tippyInstance.popper.addEventListener("mouseout", (event) => {
+    if (event.target.nodeName === "TD") {
+      const selectedRow = event.target.parentElement;
+      selectedRow.style.backgroundColor = "";
+      const activityName = selectedRow.firstElementChild.textContent;
+      const activityId = getElementIdByName(activityName)
+      //highlight activity
+      bpmnVisualization.bpmnElementsRegistry.removeCssClasses(activityId, "cause-violation")
+      // workaround for https://github.com/process-analytics/icpm-demo-2022/issues/87
+      if(complianceData.has(activityName)){
+        addRippleCircles(activityId, bpmnVisualization)
+      }
+    }
+  });

```

Comment:
**issue:** This should be done with CSS directly, as this is done in https://github.com/process-analytics/bonita-day-demo-2023/pull/10/files

Reply:
@assynour did you have time to check the review?

Reply:
I will fix it this week

Reply:
fixed in 57b6053372edb1c7c3baf5202925c300819e9f32